### PR TITLE
Add restaurant averages routes

### PR DESCRIPTION
**Pull request by steenburgh**
This is mostly philip's code - I didn't see his PR until after I created this one, but I've updated it to use a file being created by a cronjob instead of a static file. https://github.com/TheBox193/diningin-tweaks-server/pull/1

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -70,6 +70,25 @@ app.get('/rest/:id', function(req, res){
   });
 });
 
- // var server = app.listen(3000, function(){
- //   console.log('Magic is happening on port 3000')
- // });
\ No newline at end of file
+# This file is maintained by a script running via a cronjob
```
Comment (steenburgh) [inline app.js]: The code that maintains this file is located in `/home/time/diningin-time-tracker/` on our server box. This code currently lives in it's own(currently private) repo on github but can be made public or added to this repo if necessary.

File: app.js [hunk 2]
```diff
@@ -70,6 +70,25 @@ app.get('/rest/:id', function(req, res){
...
+const averagesFileLocation = '/home/time/diningin-time-tracker/averages.json';
```
Comment (steenburgh) [inline app.js:74]: The code that maintains this file is located in `/home/time/diningin-time-tracker/` on our server box. This code currently lives in it's own(currently private) repo on github but can be made public or added to this repo if necessary.

The code is run weekly via a cronjob.

**Comment by TheBox193** [comment]
Sorry for the delay, didn't see your PR. @mentions get my attention though 😸 

Yeah, let's add the code over here, that'd be cool 👍 

Want to get this rebased? I merged philip's PR.

**Comment by steenburgh** [comment]
@TheBox193 I fixed it(finally). I need your help setting up a cronjob that will automatically run our python script however. I set something up but it doesn't appear to be working.

**Pull request merged**